docs/src/main/sphinx/develop/connectors.md

```diff
@@ -671,7 +671,7 @@ The built-in SQL data types use different Java types as carrier types.
   * - ``BIGINT``
     - ``long``
   * - ``REAL``
-    - ``double``
+    - ``long``
```

Comment:
That's true but requires explanation how the real value gets encoded into the long value.

Reply:
Wouldn't `RecordCursor.getDouble()` be used here? Or both are supported?

Reply:
AFAICT `RecordCursor.getLong()` is called for `REAL`.

Reply:
As for documenting the encoding, yes that would be great. I've only looked at other connectors to figure it out, for example:

- [StandardColumnMappings](https://github.com/trinodb/trino/blob/f92b119a0ea6077dfbd72afca8dbde167f8361eb/plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/StandardColumnMappings.java#L151)
- [CassandraRecordCursor](https://github.com/trinodb/trino/blob/f92b119a0ea6077dfbd72afca8dbde167f8361eb/plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraRecordCursor.java#L126)

Reply:
You're right. `java.lang.Float.floatToRawIntBits` can be used to encode a float value into an int value. There's already a paragraph after this table explaining how to encode timestamps. Can you add a paragraph there about floats?

Reply:
Sure, I'll add that.

Reply:
Added description of `long` encoding.